app.js

```diff
@@ -16,6 +17,12 @@ app.use((err, req, res, next) => {
     }
 })
 
+app.use((err, req, res, next) => {
+    if(err.code === '22P02') {
+        res.status(400).send({msg: 'Bad Request'})
+    }
```

Comment:
You'll need to add a:
` else {
  next(err)
}`
here if you want to see any other errors in your console.log on line 27!

Reply:
done